### DISCUSSION TITLE
Шейдер "models_lfo_light_dot_weapons"

### DISCUSSION THREAD

**Discussion by Gena45hp** [OGSR/OGSR-Engine#413] [General]
Доброго дня, может кто знает, как адаптировать шейдер "models_lfo_light_dot_weapons" динамических коллиматорных прицелов из anomaly 1.5.1 к движку ogsr?

**Comment by I-AmDead** [comment]
Для этого нужно добавить один параметр в движок

**Reply by Gena45hp** [reply]
Что за параметр если не секрет?

**Reply by Gena45hp** [reply]
Этот слот в конфигах тоже где то надо будет прописывать?

**Reply by I-AmDead** [reply]
Слот то зачем?

**Reply by Gena45hp** [reply]
Или оно так заработает? немного не понимаю

**Reply by I-AmDead** [reply]
Вот этот параметр добавь и всё.
![image](https://user-images.githubusercontent.com/47627453/159837508-0899c7de-7931-4f6a-baed-1f2b6e4f6f4d.png)

**Reply by Gena45hp** [reply]
Благодарю, буду пробовать)))

**Comment by xrSimpAI** [comment]
В актуальный релиз движка этот шейдер добавлен и работает.
